src/bilder/images/vault/deploy.py

```diff
@@ -51,18 +57,22 @@
 # Set up configuration objects
 set_env_secrets(Path("consul/consul.env"))
 # Install Caddy
-caddy_config = CaddyConfig(
-    plugins=[
-        CaddyPlugin(
-            repository="github.com/caddy-dns/route53",
-            version=VERSIONS["caddy_route53"],
+traefik_config = TraefikConfig(
+    static_configuration=traefik_static.TraefikStaticConfig.parse_obj(
+        yaml.safe_load(
+            FILES_DIRECTORY.joinpath("traefik", "static_config.yaml").read_text()
```

Comment:
Did you omit these two files from the commit by accident or am I misunderstanding how the component works?